amas.md

```diff
@@ -0,0 +1,11 @@
+# Instructor "Ask Me Anything" Sessions
+
+Take a break from your project, and ask the instructors anything that's been on your mind, from "Who's your favorite Power Ranger?" to "What's the biggest challenge you've faced as a programmer?" to "What's the meaning of life?" Seriously...ask us anything!
+
+## Schedule
+
+| When | Who | Where |
+|--------|---------------------|
+| Monday, 1:30-2:30pm | Nick, Adam & Adrian | Classroom 5 |
+| Wednesday, 1:30-2:30pm | John, Jesse & Andy | " |
+| Friday, 1:30-2:30pm | Erica, Matt & Robin | " |
```

Comment:
Starting to wonder if we should do 2 sessions, with 4 & 5 instructors respectively? Just so students only have to go to 2 to hear from all of us.